### DISCUSSION TITLE
hope fcitx5-diagnose show other fcitx software version  information.

### DISCUSSION THREAD

**Discussion by chenshijie-uos** [fcitx/fcitx5#372] [Q&A] [answered]
In diagnosis of solstice display other Fcitx related software version information, such as kcm-fcitx5 Fcitx5-config-qt, fcitx5-rime, fcitx5-sogoupinyin

**Comment by wengxt** [comment, marked as answer]
https://github.com/fcitx/fcitx5/issues/373

Why you create this as a discussion?..

**Reply by chenshijie-uos** [reply]
I saw that bugs were reported in the issue, so I put it here. Diagnosis is still very useful. In the future, I hope to incorporate the logs of third-party input methods. So I hope there is a place to discuss it.

**Comment by chenshijie-uos** [comment]
curl -O https://raw.githubusercontent.com/fcitx/fcitx5/master/data/fcitx5-diagnose.sh && mv ./fcitx5-diagnose.sh ./fcitx5-diagnose && chmod 777 ./fcitx5-diagnose && ./fcitx5-diagnose

我在想能不能和您用一套，并且用户直接可以获取最新的诊断脚本。

其间出现了一个raw.githubusercontent.com域名污染问题。

https://cloud.tencent.com/developer/article/1684772

**Reply by wengxt** [reply]
这个文件不能直接引用的，里面有编译时替换的内容

**Reply by chenshijie-uos** [reply]
您觉得有没有必要这样做那？
现在碰到的问题是基于fcitx4输入法框架生态的输入问题，每个输入法厂家都用不同方式收集日志。
经常有不同版本输入法功能互相冲突，但是新版本又不存在问题了。
所以建议fcitx5生态输入法，大家都在这个脚本里面添加判断收集需要的日志信息。
